addons/account/models/account_move.py

```diff
@@ -3737,9 +3737,13 @@ def _reconcile_reversed_moves(self, reverse_moves, move_reverse_cancel):
         for move, reverse_move in zip(self, reverse_moves):
             group = (move.line_ids + reverse_move.line_ids) \
                 .filtered(lambda l: not l.reconciled) \
+                .sorted(lambda l: l.display_type != 'payment_term') \
```

Comment:
```suggestion
                .sorted(lambda l: l.account_type in ('asset_receivable', 'liability_payable')) \
```
because 'payment_term' is only for invoices and the caba could be used on something else